<ui version="4.0" >
 <author></author>
 <comment></comment>
 <exportmacro></exportmacro>
 <class>EditEntryDialog</class>
 <widget class="QDialog" name="EditEntryDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>493</width>
    <height>526</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy>
    <hsizetype>0</hsizetype>
    <vsizetype>0</vsizetype>
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize" >
   <size>
    <width>493</width>
    <height>526</height>
   </size>
  </property>
  <property name="maximumSize" >
   <size>
    <width>493</width>
    <height>526</height>
   </size>
  </property>
  <property name="windowTitle" >
   <string>Eintrag bearbeiten</string>
  </property>
  <property name="modal" >
   <bool>true</bool>
  </property>
  <widget class="QLineEdit" name="Edit_Title" >
   <property name="geometry" >
    <rect>
     <x>111</x>
     <y>90</y>
     <width>370</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="Edit_Password_w" >
   <property name="geometry" >
    <rect>
     <x>111</x>
     <y>200</y>
     <width>330</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="Edit_UserName" >
   <property name="geometry" >
    <rect>
     <x>111</x>
     <y>120</y>
     <width>370</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="Edit_Password" >
   <property name="geometry" >
    <rect>
     <x>111</x>
     <y>180</y>
     <width>330</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="Combo_Group" >
   <property name="geometry" >
    <rect>
     <x>110</x>
     <y>60</y>
     <width>220</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="Combo_IconPicker" >
   <property name="geometry" >
    <rect>
     <x>431</x>
     <y>60</y>
     <width>50</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="textLabel5" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>120</y>
     <width>90</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text" >
    <string>Benutzername:</string>
   </property>
  </widget>
  <widget class="QLabel" name="textLabel8" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>200</y>
     <width>97</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text" >
    <string>Passwort Wdhlg.:</string>
   </property>
  </widget>
  <widget class="QLabel" name="textLabel4" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>90</y>
     <width>90</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text" >
    <string>Titel:</string>
   </property>
  </widget>
  <widget class="QLabel" name="textLabel7" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>150</y>
     <width>90</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text" >
    <string>URL:</string>
   </property>
  </widget>
  <widget class="QLabel" name="textLabel6" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>180</y>
     <width>100</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text" >
    <string>Passwort:</string>
   </property>
  </widget>
  <widget class="QLabel" name="textLabel9" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>225</y>
     <width>90</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text" >
    <string>Qualität:</string>
   </property>
  </widget>
  <widget class="QLabel" name="textLabel10" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>250</y>
     <width>100</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text" >
    <string>Kommentar:</string>
   </property>
  </widget>
  <widget class="QLabel" name="textLabel11" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>410</y>
     <width>100</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text" >
    <string>gültig bis:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Edit_URL" >
   <property name="geometry" >
    <rect>
     <x>111</x>
     <y>150</y>
     <width>370</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="textLabel3" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>60</y>
     <width>90</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text" >
    <string>Gruppe:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ButtonCancel" >
   <property name="geometry" >
    <rect>
     <x>380</x>
     <y>497</y>
     <width>104</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text" >
    <string>Abbre&amp;chen</string>
   </property>
   <property name="shortcut" >
    <string>Alt+C</string>
   </property>
  </widget>
  <widget class="QLabel" name="Label_AttachmentSize" >
   <property name="geometry" >
    <rect>
     <x>110</x>
     <y>461</y>
     <width>120</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text" >
    <string>5,30 MB</string>
   </property>
  </widget>
  <widget class="QLabel" name="Banner" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>493</width>
     <height>50</height>
    </rect>
   </property>
   <property name="sizePolicy" >
    <sizepolicy>
     <hsizetype>7</hsizetype>
     <vsizetype>5</vsizetype>
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="scaledContents" >
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="textLabel1" >
   <property name="geometry" >
    <rect>
     <x>400</x>
     <y>60</y>
     <width>27</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text" >
    <string>Icon:</string>
   </property>
  </widget>
  <widget class="QLabel" name="Label_Bits" >
   <property name="geometry" >
    <rect>
     <x>400</x>
     <y>228</y>
     <width>44</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text" >
    <string>120 Bits</string>
   </property>
  </widget>
  <widget class="QToolButton" name="ButtonGenPw" >
   <property name="geometry" >
    <rect>
     <x>450</x>
     <y>200</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text" >
    <string>Ge&amp;n.</string>
   </property>
  </widget>
  <widget class="QToolButton" name="ButtonEchoMode" >
   <property name="geometry" >
    <rect>
     <x>450</x>
     <y>180</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text" >
    <string>...</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ButtonOK" >
   <property name="geometry" >
    <rect>
     <x>270</x>
     <y>497</y>
     <width>103</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text" >
    <string>O&amp;K</string>
   </property>
   <property name="shortcut" >
    <string>Alt+K</string>
   </property>
  </widget>
  <widget class="Q3TextEdit" name="Edit_Comment" >
   <property name="geometry" >
    <rect>
     <x>111</x>
     <y>250</y>
     <width>370</width>
     <height>151</height>
    </rect>
   </property>
  </widget>
  <widget class="QDateTimeEdit" name="DateTime_Expire" >
   <property name="geometry" >
    <rect>
     <x>110</x>
     <y>410</y>
     <width>194</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QCheckBox" name="CheckBox_ExpiresNever" >
   <property name="geometry" >
    <rect>
     <x>320</x>
     <y>410</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text" >
    <string>läuft nicht ab</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="Progress_Quali" >
   <property name="geometry" >
    <rect>
     <x>110</x>
     <y>227</y>
     <width>281</width>
     <height>15</height>
    </rect>
   </property>
   <property name="value" >
    <number>24</number>
   </property>
   <property name="textVisible" >
    <bool>false</bool>
   </property>
   <property name="orientation" >
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="textLabel12" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>440</y>
     <width>90</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text" >
    <string>Anhang:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Edit_Attachment" >
   <property name="enabled" >
    <bool>false</bool>
   </property>
   <property name="geometry" >
    <rect>
     <x>110</x>
     <y>440</y>
     <width>280</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolButton" name="ButtonOpenAttachment" >
   <property name="geometry" >
    <rect>
     <x>390</x>
     <y>440</y>
     <width>30</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text" >
    <string/>
   </property>
   <property name="icon" >
    <iconset/>
   </property>
  </widget>
  <widget class="QToolButton" name="ButtonSaveAttachment" >
   <property name="geometry" >
    <rect>
     <x>420</x>
     <y>440</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text" >
    <string/>
   </property>
   <property name="icon" >
    <iconset/>
   </property>
  </widget>
  <widget class="QToolButton" name="ButtonDeleteAttachment" >
   <property name="geometry" >
    <rect>
     <x>450</x>
     <y>440</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text" >
    <string/>
   </property>
   <property name="icon" >
    <iconset/>
   </property>
  </widget>
  <widget class="Line" name="line1" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>476</y>
     <width>490</width>
     <height>16</height>
    </rect>
   </property>
   <property name="frameShape" >
    <enum>QFrame::HLine</enum>
   </property>
   <property name="frameShadow" >
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="orientation" >
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <pixmapfunction></pixmapfunction>
 <customwidgets>
  <customwidget>
   <class>Q3TextEdit</class>
   <extends></extends>
   <header>q3textedit.h</header>
   <container>0</container>
   <pixmap></pixmap>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>Combo_Group</tabstop>
  <tabstop>Combo_IconPicker</tabstop>
  <tabstop>Edit_Title</tabstop>
  <tabstop>Edit_UserName</tabstop>
  <tabstop>Edit_URL</tabstop>
  <tabstop>Edit_Password</tabstop>
  <tabstop>Edit_Password_w</tabstop>
  <tabstop>Edit_Comment</tabstop>
  <tabstop>Edit_Attachment</tabstop>
  <tabstop>ButtonOpenAttachment</tabstop>
  <tabstop>ButtonSaveAttachment</tabstop>
  <tabstop>ButtonDeleteAttachment</tabstop>
  <tabstop>ButtonEchoMode</tabstop>
  <tabstop>ButtonGenPw</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
